activiti-core/activiti-spring-boot-starter/src/test/resources/application-with-candidate-starters-enabled.properties

```diff
@@ -0,0 +1,2 @@
+spring.application.name=candidate-starters-app
+spring.activiti.candidateStarter.enabled=true
```

Comment:
Wouldn't be better to remove the `spring` prefix? The fact that we are using Spring is just an implementation detail.